.ci/run-tests

```diff
@@ -1,15 +1,14 @@
 #!/usr/bin/env bash
 #
-# Version 1.1
-# - Moved to .ci folder and seperated out `run-repository.sh`
-# - Add `$RUNSCRIPTS` env var for running opensearch dependent products
+# Entrypoint to run integration tests
 
 # Default environment variables
 export TEST_SUITE="${TEST_SUITE:=oss}"
 export PYTHON_VERSION="${PYTHON_VERSION:=3.9}"
 export PYTHON_CONNECTION_CLASS="${PYTHON_CONNECTION_CLASS:=Urllib3HttpConnection}"
 export CLUSTER="${1:-opensearch}"
 export SECURE_INTEGRATION="${2:-false}"
+export OPENSEARCH_VERSION="${3:-latest}"
```

Comment:
nit: CLUSTER_VERSION

Reply:
I named it `OPENSEARCH_VERSION` since this parameter is not applicable for opendistro. WDYT?